sklearn_numba_dpex/kmeans/tests/test_kmeans.py

```diff
@@ -490,3 +491,15 @@ def test_error_raised_on_invalid_group_sizes():
             work_group_size,
             dtype,
         )
+
+
+def test_get_nb_distinct_clusters_kernel():
+    labels = [0, 1, 0, 2, 2, 7, 6, 5, 3, 3]  # NB: all values up to 7 except 4
+    expected_nb_distinct_clusters = 6
+
+    n_clusters = max(labels)
+    labels = dpt.asarray(labels, dtype=np.int32)
+
+    actual_nb_distinct_clusters = int(get_nb_distinct_clusters(labels, n_clusters))
+
+    assert actual_nb_distinct_clusters == expected_nb_distinct_clusters
```

Comment:
Thanks!

I guess we could test on randomly generated data and by composing `len` with `np.unique`, but we needn't: this is sufficient IMO.